build/packer.json

```diff
@@ -92,6 +96,71 @@
         "sudo chmod +x /opt/dxe/airtable/run_generate_chapter_data.sh",
         "/opt/dxe/airtable/run_generate_chapter_data.sh"
       ]
+    },
+
+    {
+      "type": "file",
+      "source": "airtable/generate_members_data.py",
+      "destination": "/opt/dxe/airtable/generate_members_data.py"
+    }, {
+      "type": "file",
+      "source": "airtable/run_generate_members_data.sh",
+      "destination": "/opt/dxe/airtable/run_generate_members_data.sh"
+    }, {
+      "type": "file",
+      "source": "attend/attend.css",
+      "destination": "/var/www/attend/attend.css"
+    }, {
+      "type": "file",
+      "source": "attend/attend.js",
+      "destination": "/var/www/attend/attend.js"
+    }, {
+      "type": "file",
+      "source": "attend/attend.php",
+      "destination": "/var/www/attend/attend.php"
+    }, {
+      "type": "file",
+      "source": "attend/jquery-1.11.3.min.js",
+      "destination": "/var/www/attend/jquery-1.11.3.min.js"
```

Comment:
Actually one point about pointing to their version is if they move it or reorg their site or whatever, we have to scramble.  And that situation would be crappy...  So I'm inclined to leave it.